File: internal/content/testdata/bench/csharp/rotate-matrix-cw-90.cs
public class Solution {
    public long[][] rotate_matrix_cw(long[][] matrix) {
        int n = matrix.Length;
        long[][] result = new long[n][];
        for (int i = 0; i < n; i++) {
            result[i] = new long[n];
            for (int j = 0; j < n; j++) {
                result[i][j] = matrix[n - 1 - j][i];
            }
        }
        return result;
    }
}

File: internal/content/testdata/bench/go/rotate-matrix-cw-90.go
func rotate_matrix_cw(matrix [][]int) [][]int {
	n := len(matrix)
	result := make([][]int, n)
	for i := range result {
		result[i] = make([]int, n)
		for j := 0; j < n; j++ {
			result[i][j] = matrix[n-1-j][i]
		}
	}
	return result
}
